numpy/lib/polynomial.py

```diff
@@ -423,6 +424,19 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
     cov : bool, optional
         Return the estimate and the covariance matrix of the estimate
         If full is True, then cov is not returned.
+    absolute_weights: bool, optional
```

Comment:
I realize it amount to bike-shedding, but I find this name confusing, since I've never encountered this term. If we stick close to this, I'd very much prefer a `relative_weigths=True` since I think that more clearly indicates that there is something weird about the weights.

But really what this does is forcing the reduced chi2 to unity, so maybe that is what the parameter name should reflect? Indeed, in the actual code, the weights are not used at all. Now `force_redchi2_to_unity` is a bit long...  Maybe `rescale_covariance=True`? Or just `cov_scale` or `scale_cov`?

Reply:
Just to explain the current choice of `absolute_weights`: It was suggested by @josef-pkt on the [mailing list](http://numpy-discussion.10968.n7.nabble.com/Inconsistent-results-for-the-covariance-matrix-between-scipy-optimize-curve-fit-and-numpy-polyfit-td45582.html). It is the analogue to `scipy.optimize.curve_fit`'s `absolute_sigma` parameter. Its name was decided on in the lengthy discussion within [this PR](https://github.com/scipy/scipy/pull/448#issuecomment-19677564), which was continued in [this PR](https://github.com/scipy/scipy/pull/3098). At least I somewhat like the analogy to the `curve_fit` terminology but I don't know whether this really is a valid argument here.

Reply:
p.s. In the docstring proper, be careful with single back quotes - with those, there should be an actual link target, i.e., something like `False` works because it links to the python API.

Reply:
What about this suggestion of using `cov` to indicate whether or not scaling should be done?

Reply:
To be honest, I like the simple `relative_weight` better. Like @jotasi already mentioned, it behaves like `absolute_sigma` in `curve_fit`. Still, if you insist, I can implement your proposal. In this case it would be nice, if you could point me to another function with a similar parameter, so I can have a look what type of parameter check is performed.